jinseok/routes/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const router = express.Router();
+
+const userRouter = require('./userRouter');
+const postRouter = require('./postRouter');
+const likeRouter = require('./likeRouter');
```

Comment:
```suggestion
const express = require('express');

const userRouter = require('./userRouter');
const postRouter = require('./postRouter');
const likeRouter = require('./likeRouter');

const router = express.Router();
```
필요한 모듈을 불러오고 이를 사용하는 순으로 코드를 작성해주세요.